Makefile

```diff
@@ -46,6 +46,8 @@ TARBALL:=node-problem-detector-$(VERSION).tar.gz
 # IMAGE is the image name of the node problem detector container image.
 IMAGE:=$(REGISTRY)/node-problem-detector:$(TAG)
 
+BUILD_TAGS?=""
```

Comment:
Nit: Move this line to line 63 to be together with other build tag manipulations.

Reply:
Sounds good, done.